lxd-migrate/main_migrate.go

```diff
@@ -160,7 +160,7 @@ func (c *cmdMigrate) askServer() (lxd.InstanceServer, string, error) {
 	type AuthMethod int
 
 	const (
-		authMethodCandid AuthMethod = iota
+		_ AuthMethod = iota
```

Comment:
Do we need to keep the old entry at all? Are the IDs used in a DB/config file or map to an API request ID?

Reply:
I don't think so. I left it to keep things consistent. Except now I realise that the integer value picked when running interactively will vary depending on the available authentication methods, so it makes no odds.

I'll remove it.

Reply:
Done.